.travis.yml

```diff
@@ -1,6 +1,6 @@
  sudo: false
 
  language: node_js
- node_js: "4.2"
- 
+ node_js: "10.17"
```

Comment:
I'd suggest using node 12 as we are moving toward 12 everywhere

Reply:
I did just now